packages/camera/camera/CHANGELOG.md

```diff
@@ -22,7 +26,7 @@
 
 ## 0.9.4+19
 
-* Migrate deprecated Scaffold SnackBar methods to ScaffoldMessenger.
+* Migrates deprecated Scaffold SnackBar methods to ScaffoldMessenger.
```

Comment:
We don't generally change already-published changelog entries, even if they had style issues.